dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataSqlQueryFactoryBuilder.java

```diff
@@ -0,0 +1,288 @@
+        "L.FirstStepTime AT TIME ZONE INTERVAL '0:00' HOUR TO MINUTE AS \"FirstStepTime\"",
+        "L.FlexThrottle",
+        "L.ImpactSpool",
+        "L.InternalRequestNum",
+        "L.IODecayLevel",
+        "L.IterationCount",
+        "L.KeepFlag",
+        "L.LastRespTime",
+        "L.LockDelay",
+        "L.LockLevel",
+        "L.LogicalHostID",
+        "L.LogonDateTime AT TIME ZONE INTERVAL '0:00' HOUR TO MINUTE AS \"LogonDateTime\"",
+        "L.LogonSource",
+        "L.LSN",
+        "L.MaxAMPCPUTime",
+        "L.MaxAMPCPUTimeNorm",
+        "L.MaxAmpIO",
+        "L.MaxCPUAmpNumber",
+        "L.MaxCPUAmpNumberNorm",
+        "L.MaxIOAmpNumber",
+        "L.MaxNumMapAMPs",
+        "L.MaxOneMBRowSize",
+        "L.MaxStepMemory",
+        "L.MaxStepsInPar",
+        "L.MinAmpCPUTime",
+        "L.MinAmpCPUTimeNorm",
+        "L.MinAmpIO",
+        "L.MinNumMapAMPs",
+        "L.MinRespHoldTime",
+        "L.NumFragments",
+        "L.NumOfActiveAMPs",
+        "L.NumRequestCtx",
+        "L.NumResultOneMBRows",
+        "L.NumResultRows",
+        "L.NumSteps",
+        "L.NumStepswPar",
+        "L.ParamQuery",
+        "L.ParserCPUTime",
+        "L.ParserCPUTimeNorm",
+        "L.ParserExpReq",
+        "L.PersistentSpool",
+        "L.ProcID",
+        "L.ProfileID",
+        "L.ProfileName",
+        "L.ProxyRole",
+        "L.ProxyUser",
+        "L.ProxyUserID",
+        "L.QueryBand",
+        "L.QueryRedriven",
+        "L.QueryText",
+        "L.ReDriveKind",
+        "L.RemoteQuery",
+        "L.ReqIOKB",
+        "L.ReqPhysIO",
+        "L.ReqPhysIOKB",
+        "L.RequestMode",
+        "L.RequestNum",
+        "L.SeqRespTime",
+        "L.SessionID",
+        "L.SessionTemporalQualifier",
+        "L.SpoolUsage",
+        "L.StartTime AT TIME ZONE INTERVAL '0:00' HOUR TO MINUTE AS \"StartTime\"",
+        "L.StatementGroup",
+        "L.Statements",
+        "L.StatementType",
+        "L.SysDefNumMapAMPs",
+        "L.TacticalCPUException",
+        "L.TacticalIOException",
+        "L.TDWMEstMemUsage",
+        "L.ThrottleBypassed",
+        "L.TotalFirstRespTime",
+        "L.TotalIOCount",
+        "L.TotalServerByteCount",
+        "L.TTGranularity",
+        "L.TxnMode",
+        "L.TxnUniq",
+        "L.UnitySQL",
+        "L.UnityTime",
+        "L.UsedIota",
+        "L.UserID",
+        "L.UserName",
+        "L.UtilityByteCount",
+        "L.UtilityInfoAvailable",
+        "L.UtilityRowCount",
+        "L.VHLogicalIO",
+        "L.VHLogicalIOKB",
+        "L.VHPhysIO",
+        "L.VHPhysIOKB",
+        "L.WarningOnly",
+        "L.WDName"
+      };
+
+  private ZonedInterval interval;
+  private String logTable;
+  private String queryTable;
+  private List<String> conditions;
+  private List<String> orderColumns;
+  private List<String> expressions;
+  private boolean isAssessment;
+  // For Terdata version <= 14
+  private boolean isOldVersion;
+  private final boolean isLSql;
```

Comment:
Correct me if I'm wrong, but the 2 fields above (`isOldVersion` and `isLSql`) would not be needed, if the class `Teradata14LogsConnector.java` was reverted. I assume the behavior for Teradata<=14 would not change.

Advantages:
- less effort now
- `EXPRESSIONS_LSQL_TBL` and `EXPRESSIONS_LOG_TBL` constants and `enumNames` method could be removed from this class
- any code related to the 2 boolean fields (above) could also be simplified
- there would be no need to add tests for Teradata<=14 now

Also, if in the future it turns out that Teradata<=14 is needed, we will still be able to change the code of the old connector in a separate PR.